### ISSUE TITLE
Error in core\model\phpthumb\phpthumb.class.php 990

### ISSUE THREAD

**Issue by ghoulru** [modxcms/revolution#14489]
### Summary
Notice: Use of undefined constant IMG_WEBP - assumed 'IMG_WEBP' in \core\model\phpthumb\phpthumb.class.php on line 990

### Step to reproduce
Rename "core" and "manager" directories/
Create TV named "image". Media sources: default and custom, same problem.
don't show images thumbnails in /manager/

### Observed behavior
If i comment or delete code from line 990
```
if ($imagetypes & IMG_WEBP) {
$this->thumbnailFormat         = 'webp';
				$AvailableImageOutputFormats[] = 'webp';
			}
```
everythig is ok.

screenshots
[error in code](http://перекраска-дома.рф/phpthumb_modx.png)
[no thumbnail](http://перекраска-дома.рф/modx_manager.png)

### Environment
MODX 2.7.1, php 7.0.8 , mysql 5.5, Fitefox 65.0.2. Using openserver 5.2.2

**Comment by JoshuaLuckers** [comment]
The constant `IMG_WEBP` is available as of PHP 5.6.25 and PHP 7.0.10, respectively. 
Source: [PHP Predefined Constants](https://secure.php.net/manual/en/image.constants.php)

```
/**
 * Used as a return value by imagetypes() Available as of PHP 5.6.25 and PHP 7.0.10, respectively.
 * @link https://php.net/manual/en/image.constants.php
 */
define('IMG_WEBP', 32);
```

**Comment by Ibochkarev** [comment]
Fixed #14494

@modxbot close
